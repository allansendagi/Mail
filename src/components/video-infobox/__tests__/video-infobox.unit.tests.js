import {VideoInfoBox} from '../video-infobox.component';
import {shallow, configure} from 'enzyme';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});
describe('VideoInfoBox', () => {
  test('renders collapsed', () => {
    const wrapper = shallow(<VideoInfoBox/>);
    expect(wrapper).toMatchSnapshot();
  });
  test('renders expanded', () => {
    const wrapper = shallow(<VideoInfoBox/>);
    wrapper.setState({collapsed: false});
    expect(wrapper).toMatchSnapshot();
  });
});