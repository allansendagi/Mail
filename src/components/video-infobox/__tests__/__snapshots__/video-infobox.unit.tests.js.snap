// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoInfoBox renders collapsed 1`] = `ShallowWrapper {}`;

exports[`VideoInfoBox renders expanded 1`] = `ShallowWrapper {}`;
